styles/styles.css

```diff
@@ -459,9 +554,13 @@
     --col-width: calc(100% / 12);
   }
 
-  /*[class*="col-"] {*/
```

Comment:
also, remove

Reply:
you talk about comments that already was removed in this commit or you don't like 554th line?

Reply:
About comment